doc/index.rst

```diff
@@ -173,3 +173,4 @@ Strawberry Fields is **free** and **open source**, released under the Apache Lic
    code/gbs/g_sample
    code/gbs/clique
    code/gbs/similarity
+   code/gbs/pp
```

Comment:
Have we agreed anywhere to name the module pp? I don't like this much, 'pee pee' makes me think of somehting different than point processes.
Maybe `points`?
`point_processes`?

Reply:
I will change it to points.

Reply:
I like the sound of pp samples 😁
But seriously, yeah I agree `pp` isn't great

`points` is good... what about `point`?

Reply:
PS, when updating the module name, also update the tests and docs

Reply:
I like `points` because it evokes a collection of points, which is what point processes produce. With `point` it makes me think, what's the point? :wink:

Reply:
I guess my preference for "point" stems from it being shortened from "point_process".

But for the sake of not holding things up, I'm happy for `points`